### PR TITLE
Add public key - keyLength info into Manage Keys dialog

### PR DESCRIPTION
**Pull request by tuanta**
Add public key - keyLength info into Manage Keys dialog.

### CODE REVIEW

**Comment by barrydegraaff** [comment]
Thanks again!

**Comment by barrydegraaff** [comment]
I am afraid, this is one of those cases where a `simple` request gets complicated

The key length that is displayed, is the key length of the primary key... this may or may not be the key that is used for encryption...

Here is my 2006 DSA key:
http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=barrydegraaff&fingerprint=on
	 Fingerprint=97F4 694A 1D9A EDAD 0125  33DB 725D DD15 6D36 A2D0 
sub  4096g/A4DDD401 2006-11-01  

Documentation:
http://www.macfreek.nl/memory/Analysing_PGP_Identities

gpg --with-colons --fixed-list-mode --with-fingerprint --list-keys A4DDD401
tru::1:1424969286:0:3:1:5
pub:-:1024:17:725DDD156D36A2D0:1162401609:::-:::scESC:
fpr:::::::::97F4694A1D9AEDAD012533DB725DDD156D36A2D0:
uid:-::::1162401609::3B352CA7B445E9F6A2A23B433D82B25958FF5F6E::Barry de Graaff (www.barrydegraaff.tk) <info@barrydegraaff.tk>:
sub:-:4096:16:AF8B1052A4DDD401:1162401859::::::e:

Basically means, signing is 1024 and encryption is 4096...

Not sure what we can do about this... but if there was a way to determine what key is
used for encryption, then ideally we would like to display that key length.
